docs/docs.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+
+# Get the 1st argument passed to the script and convert it to lowercase
+arg1=$(echo $1 | tr '[:upper:]' '[:lower:]')
+arg2=$(echo $2 | tr '[:upper:]' '[:lower:]')
+
+
+# Build the MkDocs container which will be used to build/serve the MkDocs project
+build_container () {
+    echo "Building the container..."
+    docker build -t mkdocs:latest ./mkdocs/
```

Comment:
Let's not use `mkdocs` as a tag, it might conflict with others. Perhaps `drift_docs_builder`?

Reply:
Done